values.go

```diff
@@ -36,7 +36,7 @@ import (
 // Value
 
 type Value interface {
-	isValue()
```

Comment:
Why does this need to be exported? Having it private means that no additional types conforming to Value can be declared outside the package.